### DISCUSSION TITLE
Is the command allvariables still valid?

### DISCUSSION THREAD

**Discussion by P11GH** [yalmip/YALMIP#1254] [Possible bug or missing functionality]
I was trying to use the command "allvariables" in YALMIP (https://yalmip.github.io/command/allvariables/) and copied the exmaple in the webpage to MATLAB, but it returns the error "funciton or variable allvariables unrecognized" (translated from the error information in the picture below), as shown in the picture below. Does anyone knows what's wrong here?
<img width="381" alt="image" src="https://user-images.githubusercontent.com/50893243/228233293-dcb3398f-d265-4277-bfe1-c04bb9fd5d58.png">

**Comment by johanlofberg** [comment]
Requires the develop branch

**Reply by P11GH** [reply]
OK, thank you.
